src/modules/router/router.resolver.ts

```diff
@@ -154,6 +163,41 @@ export class RouterResolver {
         return this.routerService.getAllPairs(page.offset, page.limit, filter);
     }
 
+    @Query(() => PairsResponse)
+    async filterablePairs(
```

Comment:
the result for this query should be a list with `filteredPairs`. Might be a better name for the query

Reply:
done